spec/support/helpers.rb

```diff
@@ -347,6 +347,15 @@ def simulate_platform(platform)
       ENV["BUNDLER_SPEC_PLATFORM"] = old if block_given?
     end
 
+    def simulate_ruby_version(version)
+      return if version == RUBY_VERSION
+      old = ENV["BUNDLER_SPEC_RUBY_VERSION"]
```

Comment:
Scratch that, seems like `remove_const` is private so that wouldn't work. Never mind!